src/security/PeerConnector.cc

```diff
@@ -763,16 +763,23 @@ Security::PeerConnector::resumeNegotiation()
     auto lastError = suspendedError_; // may be reset below
     suspendedError_ = nullptr;
 
+    negotiateAfterRevalidateCertificates(*lastError);
+}
+
+void
+Security::PeerConnector::negotiateAfterRevalidateCertificates(const Security::IoResult &lastError)
```

Comment:
We are inside `Security` already:

```suggestion
Security::PeerConnector::negotiateAfterRevalidateCertificates(const IoResult &lastError)
```

Reply:
commit be5aa86a21